apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/DataLayoutStrategyGeneratorSparkAppTest.java

```diff
@@ -0,0 +1,85 @@
+package com.linkedin.openhouse.jobs.spark;
+
+import com.linkedin.openhouse.tablestest.OpenHouseSparkITest;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.analysis.NoSuchTableException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class DataLayoutStrategyGeneratorSparkAppTest extends OpenHouseSparkITest {
```

Comment:
Should we add the test cases to https://github.com/linkedin/openhouse/blob/main/libs/datalayout/src/test/java/com/linkedin/openhouse/datalayout/generator/OpenHouseDataLayoutStrategyGeneratorTest.java

Reply:
No change in the generator itself, so no need for test cases.